### PR TITLE
refactor: github.com/alibaba-> github.com/dragonflyoss

### PR DESCRIPTION
**Pull request by liuhengloveyou**
### Ⅰ. Describe what this PR did
github.com/alibaba-> github.com/dragonflyoss

### Ⅱ. Does this pull request fix one issue?
NO

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
NO

### Ⅳ. Describe how to verify it
go build ...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,14 +66,14 @@ Since you are ready to improve Dragonfly with a PR, we suggest you could take a
 
 To put forward a PR, we assume you have registered a GitHub ID. Then you could finish the preparation in the following steps:
 
-1. **FORK** Dragonfly to your repository. To make this work, you just need to click the button Fork in right-left of [alibaba/Dragonfly](https://github.com/alibaba/Dragonfly) main page. Then you will end up with your repository in `https://github.com/<your-username>/Dragonfly`, in which `your-username` is your GitHub username.
+1. **FORK** Dragonfly to your repository. To make this work, you just need to click the button Fork in right-left of [alibaba/Dragonfly](https://github.com/dragonflyoss/Dragonfly) main page. Then you will end up with your repository in `https://github.com/<your-username>/Dragonfly`, in which `your-username` is your GitHub username.
```
Comment (allencloud) [inline CONTRIBUTING.md]: how about also updating `[alibaba/Dragonfly]`?

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
 In Alibaba, Dragonfly is invoked 2 Billion times and the data distributed is 3.4PB every month. Dragonfly has become one of the most important pieces of infrastructure at Alibaba. The reliability is up to 99.9999% (*1).
 
-While container technologies makes devops life easier most of the time, it sure brings a some challenges: the efficiency of image distribution, especially when you have to replicate image distribution on several hosts. Dragonfly works extremely well with both Docker and [PouchContainer](https://github.com/alibaba/pouch) for this scenario. It also is compatible with any other container formats.
+While container technologies makes devops life easier most of the time, it sure brings a some challenges: the efficiency of image distribution, especially when you have to replicate image distribution on several hosts. Dragonfly works extremely well with both Docker and [PouchContainer](https://github.com/dragonflyoss/pouch) for this scenario. It also is compatible with any other container formats.
```
Comment (allencloud) [inline README.md]: No, we do not need to change this part.

File: docs/overview/what_is_dragonfly.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ At Alibaba, every month Dragonfly is invoked two billion times and distributes 3
 
 While container technologies makes DevOps life easier most of the time, it surely brings some challenges: for example the efficiency of image distribution, especially when you have to replicate image distribution on several hosts.
 
-Dragonfly works extremely well with both Docker and [PouchContainer](https://github.com/alibaba/pouch) in this scenario. It's also compatible with containers of other formats. It delivers up to 57 times the throughput of native docker and saves up to 99.5% of the out bandwidth of registry.
+Dragonfly works extremely well with both Docker and [PouchContainer](https://github.com/dragonflyoss/pouch) in this scenario. It's also compatible with containers of other formats. It delivers up to 57 times the throughput of native docker and saves up to 99.5% of the out bandwidth of registry.
```
Comment (allencloud) [inline docs/overview/what_is_dragonfly.md]: No, we do not need this part of change.

File: vendor/github.com/spf13/cobra/README.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ Many of the most widely used Go projects are built using Cobra including:
 * [Nanobox](https://github.com/nanobox-io/nanobox)/[Nanopack](https://github.com/nanopack)
 * [rclone](http://rclone.org/)
 * [nehm](https://github.com/bogem/nehm)
-* [Pouch](https://github.com/alibaba/pouch)
+* [Pouch](https://github.com/dragonflyoss/pouch)
```
Comment (allencloud) [inline vendor/github.com/spf13/cobra/README.md]: No, we do not need this part of change.

**Comment by allencloud** [comment]
Thanks a lot for your work. This is exactly what we need. Just need some minor change there. @liuhengloveyou

File: README.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
+[![License](https://img.shields.io/badge/license-Apache%202-brightgreen.svg)](https://github.com/dragonflyoss/Dragonfly/blob/master/LICENSE)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Falibaba%2FDragonfly.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Falibaba%2FDragonfly?ref=badge_shield)
-[![GoDoc](https://godoc.org/github.com/alibaba/Dragonfly?status.svg)](https://godoc.org/github.com/alibaba/Dragonfly)
-[![Go Report Card](https://goreportcard.com/badge/github.com/alibaba/Dragonfly)](https://goreportcard.com/report/github.com/alibaba/Dragonfly)
+[![GoDoc](https://godoc.org/github.com/dragonflyoss/Dragonfly?status.svg)](https://godoc.org/github.com/dragonflyoss/Dragonfly)
+[![Go Report Card](https://goreportcard.com/badge/github.com/dragonflyoss/Dragonfly)](https://goreportcard.com/report/github.com/dragonflyoss/Dragonfly)
 [![Build Status](https://travis-ci.org/alibaba/Dragonfly.svg?branch=master)](https://travis-ci.org/alibaba/Dragonfly)
 [![CircleCI](https://circleci.com/gh/alibaba/Dragonfly.svg?style=svg)](https://circleci.com/gh/alibaba/Dragonfly)
```
Comment (lowzj) [inline README.md]: `[CircleCI](https://circleci.com/gh/dragonflyoss/Dragonfly)`

File: README.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
 [![codecov](https://codecov.io/gh/alibaba/Dragonfly/branch/master/graph/badge.svg)](https://codecov.io/gh/alibaba/Dragonfly)
```
Comment (lowzj) [inline README.md]: `[codecov](https://codecov.io/gh/dragonfly/Dragonfly)`

File: docs/_index.md [hunk 7]
```diff
@@ -3,10 +3,10 @@ title: "Dragonfly"
...
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Falibaba%2FDragonfly.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Falibaba%2FDragonfly?ref=badge_shield)
-[![GoDoc](https://godoc.org/github.com/alibaba/Dragonfly?status.svg)](https://godoc.org/github.com/alibaba/Dragonfly)
-[![Go Report Card](https://goreportcard.com/badge/github.com/alibaba/Dragonfly)](https://goreportcard.com/report/github.com/alibaba/Dragonfly)
+[![GoDoc](https://godoc.org/github.com/dragonflyoss/Dragonfly?status.svg)](https://godoc.org/github.com/dragonflyoss/Dragonfly)
+[![Go Report Card](https://goreportcard.com/badge/github.com/dragonflyoss/Dragonfly)](https://goreportcard.com/report/github.com/dragonflyoss/Dragonfly)
 [![Build Status](https://travis-ci.org/alibaba/Dragonfly.svg?branch=master)](https://travis-ci.org/alibaba/Dragonfly)
 [![CircleCI](https://circleci.com/gh/alibaba/Dragonfly.svg?style=svg)](https://circleci.com/gh/alibaba/Dragonfly)
 [![codecov](https://codecov.io/gh/alibaba/Dragonfly/branch/master/graph/badge.svg)](https://codecov.io/gh/alibaba/Dragonfly)
```
Comment (lowzj) [inline docs/_index.md]: 3 lines above:
```
[![Build Status](https://travis-ci.org/dragonflyoss/Dragonfly.svg?branch=master)](https://travis-ci.org/dragonflyoss/Dragonfly)
[![CircleCI](https://circleci.com/gh/dragonflyoss/Dragonfly.svg?style=svg)](https://circleci.com/gh/dragonflyoss/Dragonfly)
[![codecov](https://codecov.io/gh/dragonflyoss/Dragonfly/branch/master/graph/badge.svg)](https://codecov.io/gh/dragonflyoss/Dragonfly)
```

File: README.md [hunk 8]
```diff
@@ -1,13 +1,13 @@
...
-[![Go Report Card](https://goreportcard.com/badge/github.com/alibaba/Dragonfly)](https://goreportcard.com/report/github.com/alibaba/Dragonfly)
-[![Build Status](https://travis-ci.org/alibaba/Dragonfly.svg?branch=master)](https://travis-ci.org/alibaba/Dragonfly)
-[![CircleCI](https://circleci.com/gh/alibaba/Dragonfly.svg?style=svg)](https://circleci.com/gh/alibaba/Dragonfly)
-[![codecov](https://codecov.io/gh/alibaba/Dragonfly/branch/master/graph/badge.svg)](https://codecov.io/gh/alibaba/Dragonfly)
+[![GoDoc](https://godoc.org/github.com/dragonflyoss/Dragonfly?status.svg)](https://godoc.org/github.com/dragonflyoss/Dragonfly)
+[![Go Report Card](https://goreportcard.com/badge/github.com/dragonflyoss/Dragonfly)](https://goreportcard.com/report/github.com/dragonflyoss/Dragonfly)
+[![Build Status](https://travis-ci.org/dragonflyoss/Dragonfly.svg?branch=master)](https://travis-ci.org/dragonflyoss/Dragonfly)
+[![CircleCI](https://circleci.com/gh/dragonflyoss/Dragonfly.svg?style=svg)](https://circleci.com/gh/dragonflyoss/Dragonfly)
+[![codecov](https://codecov.io/gh/dragonfly/Dragonfly/branch/master/graph/badge.svg)](https://codecov.io/gh/dragonfly/Dragonfly)
```
Comment (lowzj) [inline README.md]: `[codecov](https://codecov.io/gh/dragonflyoss/Dragonfly)`

File: FAQ.md [hunk 9]
```diff
@@ -12,7 +12,7 @@ It aims to resolve issues related to low-efficiency, low-success rate and waste
 
 In Alibaba, Dragonfly is invoked 2 Billion times and the data distributed is 3.4PB every month. Dragonfly has become one of the most important pieces of infrastructure at Alibaba.
 
-While container technologies makes devops life easier most of the time, it sure brings a some challenges: the efficiency of image distribution, especially when you have to replicate image distribution on several hosts. Dragonfly works extremely well with both Docker and [PouchContainer](https://github.com/alibaba/pouch) for this scenario. It also is compatible with any other container formats.
+While container technologies makes devops life easier most of the time, it sure brings a some challenges: the efficiency of image distribution, especially when you have to replicate image distribution on several hosts. Dragonfly works extremely well with both Docker and [PouchContainer](https://github.com/dragonflyoss/pouch) for this scenario. It also is compatible with any other container formats.
```
Comment (lowzj) [inline FAQ.md]: no need to change here

File: README.md (hunk 2 above)
Comment (lowzj) [inline README.md]: no need to change here

File: docs/_index.md [hunk 10]
```diff
@@ -20,7 +20,7 @@ At Alibaba, every month Dragonfly is invoked two billion times and distributes 3
 
 While container technologies makes DevOps life easier most of the time, it surely brings some challenges: for example the efficiency of image distribution, especially when you have to replicate image distribution on several hosts.
 
-Dragonfly works extremely well with both Docker and [PouchContainer](https://github.com/alibaba/pouch) in this scenario. It's also compatible with containers of other formats. It delivers up to 57 times the throughput of native docker and saves up to 99.5% of the out bandwidth of registry.
+Dragonfly works extremely well with both Docker and [PouchContainer](https://github.com/dragonflyoss/pouch) in this scenario. It's also compatible with containers of other formats. It delivers up to 57 times the throughput of native docker and saves up to 99.5% of the out bandwidth of registry.
```
Comment (lowzj) [inline docs/_index.md]: no need to change here

**Comment by lowzj** [comment]
I polish the commit logs and have merged this pull request via command line. [merge commit](https://github.com/dragonflyoss/Dragonfly/commit/6fe07f67763496e36cff4ad0971acf2bf3cf01d3)

**Comment by allencloud** [comment]
Why not merge it just on website? @lowzj 
Is this because the multi commits?

**Comment by lowzj** [comment]
@allencloud Not only multi commits, it's a little mess to see commit history by using `git log --graph` if we just merge pull request on website.

merge just on website
image

via command line
image

**Pull request closed without merging**